xml/System/OperatingSystem.xml

```diff
@@ -566,7 +566,7 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Indicates whether the current application is running on Mac Catalyst.</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if the current application is running on Mac Catalyst; <see langword="false" /> otherwise.</returns>
```